quill-core/src/main/scala/io/getquill/MirrorIdiom.scala

```diff
@@ -205,6 +205,7 @@ class MirrorIdiom extends Idiom {
     case Insert(query, assignments)    => stmt"${query.token}.insert(${assignments.token})"
     case Delete(query)                 => stmt"${query.token}.delete"
     case Returning(query, alias, body) => stmt"${query.token}.returning((${alias.token}) => ${body.token})"
+    case ReturningRecord(query)        => stmt"${query.token}.returningRecord"
```

Comment:
do we need to transport the type here as well?

Reply:
Yes